packages/lesswrong/components/comments/CommentFrame.tsx

```diff
@@ -34,9 +34,9 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
   new: {
     '&&': {
-      borderLeft: `solid 5px ${theme.palette.secondary.light}`,
+      borderLeft: `solid 5px ${theme.palette.secondary.light}8c`,
       '&:hover': {
-        borderLeft: `solid 5px ${theme.palette.secondary.main}`
+        borderLeft: `solid 5px ${theme.palette.secondary.main}8c`
```

Comment:
Idk what these changes were intended to do, or how much they affect e.g. existing comments (both on LW and on other forums).  @Discordius ?